bin/test_integration_rhel

```diff
@@ -1,23 +1,59 @@
 #!/bin/bash -ex
 
+function print_help() {
+  cat << EOF
+
+Run integration tests on RHEL container
+
+Usage: test_integration_rhel [options]
+
+    -h, --help            Shows this help message.
+
+    --rhel-version=7      Used to Set the RHEL version to test (e.g --rhel-version=7).
```

Comment:
i think that the flag name should be alone and the example should go at the end (like you already have)
```suggestion
    --rhel-version      Used to Set the RHEL version to test (e.g --rhel-version=7).
```

Reply:
also - it seems that the default value is 8. can you add this to this comment?